index.bs

```diff
@@ -472,8 +470,54 @@ template contents). It consistes of these steps:
             |configuration|["{{SanitizerConfig/dataAttributes}}"] is true
       1. If |handleJavascriptNavigationUrls| and &laquo;[|elementName|, |attrName|]&raquo; matches an entry in the
          [=built-in navigating URL attributes list=], and if |attribute|'s [=protocol=] is
-         "`javascript:`":
-         1. Then remove |attribute| from |child|.
+         "`javascript:`",
+         then [=/remove=] |attribute| from |child|.
+      1. If |handleJavascriptNavigationUrls| and |child|'s [=Element/namespace=]
+         [=string/is=] the [=MathML Namespace=] and |attr|'s
+         [=Attr/local name=] [=string/is=]
+         "`href`" and  |attr|'s [=Attr/namespace=] [=string/is=] `null` or the
+         [=XLink namespace=] and |attr| [=contains a javascript: URL=],
+         then [=/remove an attribute|remove=] |attr|.
+      1. If the [=built-in animating URL attributes list=]
+         [=SanitizerConfig/contains=]
+         &laquo;[|elementName|, |attrName|]&raquo; and |attr|'s
+         [=get an attribute value|value=] [=string/is=] "`href`" or
+         "`xlink:href`", then [=/remove an attribute|remove=] |attr|.
+
+</div>
+
+
+<div class=note>
+<span class=marker>Note:</span> Current browsers support `javascript:` URLs
+only when navigating. Since navigation itself is not an XSS threat we treat
+navigation to `javascript:` URLs, but not navigations in general.
```

Comment:
This latter sentence reads a bit weird. Maybe "we handle" instead of "we treat"?

Reply:
Done.